<!DOCTYPE html>

<head>

<link rel="stylesheet" type="text/css" media="all" href="css/SiteStyle.css">
<link rel="shortcut icon" type="image/x-icon" href="media/ccpsas_icon.ico">
<title>CCP-SAS</title>

<style>
  em {
	 background-color: orange;
	 font-style:normal;
	 font-weight:bold;}
  table.covers {
     table-layout: fixed;
	 width:100%;}
  td.covers {
     padding:2px;}
</style>

</head>


<body>

<!-- ADD PAGE SUBTITLE HERE - always start HeaderTitle with <br> -->
<div id="pageHeader">
	<div id="HeaderTitle"><br>Impact</div>
</div>

<!-- ADD NEW MENU ITEM HERE AND ON ALL OTHER TOP PAGES -->
<div id="menuItem">
	<a href="index.html">home</a>
	<a href="about.html">about CCP-SAS</a> 	
	<a href="use.html">software</a> 	
	<a href="meetings.html">meetings/workshops</a>
	<a href="resources.html">reports/resources</a>
    <a href="contact.html">people</a>
    <a href="impact.html">impact</a>
</div>

<div id="textContainer">

<!-- THIS IS WHERE YOU ADD TEXT FOR MAIN BODY  
   Remember the main style sheet defines the default styles which should be overriden only after 
   careful thought. In that vein please remember to use div id="linkList" when using a bullted list
   in order to keep the styles consistant across the site (see the home page for example of use) -->

   <div id="CentralAnnouncement">
   		CCP-SAS Impact
   </div>

   <h3>CCP-SAS Covers</h3>
   <p>
   <br>
     <table class="covers">
	   <tbody>
	    <tr>
	     <td class"covers"><img src="./media/jcc24510-toc-0001-m-reduced.jpg" alt="Cover picture from  Journal of Computational Chemistry (2016) 37(29)" style="width:100%;"></td>
	     <td class"covers"><img src="./media/2021_Biophysical_journal_front_cover.jpg" alt="Cover picture from  Biophyscal Journal 200, 1814(2021)" style="width:100%;"></td>
         <td class"covers"><img src="./media/2021_jbc_front_cover.jpg" alt="Cover picture from  J. Biol Chem. 297, 100995(2021)" style="width:100%;"></td>
         <td class"covers"><img src="./media/JBC_300_1_COVER.jpg" alt="Cover picture from  J. Biol Chem. 299, 105337(2023)" style="width:100%;"></td>
<!--         <td class"covers">&nbsp;</td>
-->
	    <tr>
	     <td class"covers"><center><i><a href="http://onlinelibrary.wiley.com/doi/10.1002/jcc.24474/abstract">Monte Carlo simulation algorithm for B-DNA</a></center></td>
	     <td class"covers"><center><i><a href="https://doi.org/10.1016/j.bpj.2021.02.038">Solution structure of deglycosylated human IgG1 shows the role of CH2 glycans in its conformation.</a></i></center></td>
         <td class"covers"><center><i><a href="https://doi.org/10.1016/j.jbc.2021.100995">Solution structures of human myeloma IgG3 antibody reveal extended Fab and Fc regions relative to the other IgG subclasses.</a></i></center></td>
         <td class"covers"><center><i><a href="https://doi.org/10.1016/j.jbc.2023.105337 ">The solution structure of the heavy chain-only C5-Fc nanobody reveals exposed variable regions that are optimal for COVID-19 antigen interactions.</a></i></center></td>
<!--         <td class"covers">&nbsp;</td>
-->
	    <tr>
	   </tbody>
	 </table>


   <h3>Publications arising from CCP-SAS software projects:</h3>

   <div id="linkList">
	<ul>

        <!-- 2025 -->
        <b>2025</b>
        <li>E. Brookes, J. E. Curtis, A. Householder, and M. Rocco. SAXS-A-FOLD: a website for fast ensemble modeling optimizing the fit of AlphaFold or user-supplied protein structures with flexible regions to SAXS data. Applied Crystallography, 58(3). J. Appl. Cryst. 58(3) (2025). DOI <a href="https://doi.org/10.1107/S1600576725003590" title="article DOI link"> 10.1107/S1600576725003590</a></li>
<br>
         <!-- 2024 -->
        <b>2024</b>
        <li> X. Gao, H. Iqbal, D.-Q Yu, J. Gor, A. R.Coker, S. J. Perkins. The SCR-17 and SCR-18 glycans in human complement Factor H enhance its regulatory function.  . Biol. Chem. 300, 107624 (2024). DOI <a href="https://doi.org/10.1016/j.jbc.2024.107624" title="article DOI link"> 10.1016/j.jbc.2024.107624</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/39098532" title="pubmed link">39098532</a></li>
        <li> V. A. Spiteri, J. Doutch, R. P. Rambo, J. S. Bhatt, J. Gor, P. A. Dalby, S. J. Perkins. Using atomistic solution scattering modelling to elucidate the role of the Fc glycans in human IgG4.  PLoS ONE 19, e0300964 (2024). DOI <a href="https://doi.org/10.1371/journal.pone.0300964" title="article DOI link"> 10.1371/journal.pone.0300964</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/38557973" title="pubmed link">38557973</a></li>
        <li> S. Himbert, D. Gaboo, E. Brookes, J. F. Nagle, M. C. Rheinstader MEDUSA: A cloud-based tool for the analysis of X-ray diffuse scattering to obtain the bending modulus from oriented membrane stacks.  PLoS Comput Biol 20, e1011749 (2024). DOI <a href="https://doi.org/10.1371/journal.pcbi.1011749" title="article DOI link"> 10.1371/journal.pcbi.1011749</a></li>
<br>
         <!-- 2023 -->
        <b>2023</b>
        <li> X. Gao, J. W. Thrush, J. Gor, J. H. Naismith, R. J. Owens, S. J. Perkins. The solution structure of the heavy chain-only C5-Fc nanobody reveals exposed variable regions that are optimal for COVID-19 antigen interactions. J. Biol. Chem 299, 105337 (2023). DOI <a href="https://doi.org/10.1016/j.jbc.2023.105337" title="article DOI link"> 10.1016/j.jbc.2023.105337</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37838175" title="pubmed link">37838175</a> </li>
        <li> A. Y. Xu, M. A. Blanco, M. M. Castellanos, C. W. Meuse, K. Mattison, L. Karageorgos, H. W. Hatch, V. K. Shen, J. E. Curtis Role of Domain–Domain Interactions on the Self-Association and Physical Stability of Monoclonal Antibodies: Effect of pH and Salt. J. Phys. Chem. B 8344, 127 (2023). DOI <a href="https://doi.org/10.1021/acs.jpcb.3c0392" title="article DOI link"> 10.1021/acs.jpcb.3c0392</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37751332" title="pubmed link">37751332</a> </li>
        <li> G. K. Hui, X. Gao, J. Gor, J. Lu, P. D. Sun, S. J. Perkins. The solution structure of the unbound IgG Fc receptor CD64 resembles its crystal structure: Implications for function. PLoS ONE e0288351, 18 (2023). DOI <a href="https://doi.org/10.1371/journal.pone.0288351" title="article DOI link"> 10.1371/journal.pone.0288351</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37733670" title="pubmed link">37733670</a> </li>
        <li> A. H. Larsen, E. H. Brookes,M. C. Pedersen, J. J. K. Kirkensgaard Shape2SAS: a web application to simulate small-angle scattering data and pair distance distributions from user-defined shapes. J Appl Cryst 1287, 56 (2023). DOI <a href="https://doi.org/10.1107/S1600576723005848" title="article DOI link"> 10.1107/S1600576723005848</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37555217" title="pubmed link">37555217</a> </li>
        <li> E. H. Brookes, M. Rocco, P. Vachette, J. Trewhella AlphaFold-predicted protein structures and small-angle X-ray scattering: insights from an extended examination of selected data in the Small-Angle Scattering Biological Data Bank. J Appl Cryst 910, 56 (2023). DOI <a href="https://doi.org/10.1107/S1600576723005344" title="article DOI link"> 10.1107/S1600576723005344</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37555230" title="pubmed link">37555230</a> </li>
        <li> E. H. Brookes, M. Rocco Beyond the US-SOMO-AF database: a new website for hydrodynamic, structural, and circular dichroism calculations on user-supplied structures. Eur Biophys J 299, 102799 (2023). DOI <a href="https://doi.org/10.1007/s00249-023-01636-1" title="article DOI link"> 10.1007/s00249-023-01636-1</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36853343" title="pubmed link">36853343</a> </li>
        <li> H. Iqbal,K. F. Fung, J. Gor, A. C. Bishop, G. I. Makhatadze, B. Brodsky, S. J. Perkins. A solution structure analysis reveals a bent collagen triple helix in the complement activation recognition molecule mannan-binding lectin. J. Biol. Chem. 299, 102799 (2023). DOI <a href="https://doi.org/10.1016/j.jbc.2022.102799" title="article DOI link"> 10.1016/j.jsb.2022.107876</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36528062" title="pubmed link">36528062</a></li>
<br>
         <!-- 2022 -->
        <b>2022</b>
        <li> J. Trewhella, P. Vachette, J. Bierma, C. Blanchet, E. Brookes, S. Chakravarthy, L. Chatzimagas, T. E. Cleveland IV, N. Cowieson, B. Crossett, A. P. Duff, D. Franke, F. Gabel, R. E. Gillilan, M. Graewert, A. Grishaev, J. M. Guss, M. Hammel, J. Hopkins, Q. Huang, J. S. Hub, G. L. Hura, T. C. Irving, C. M. Jeffries, C. Jeong, N. Kirby, S. Krueger, A. Martel, T. Matsui, N. Li, J. Pérez, L. Porcar, T. Prangé, I. Rajkovic, M. Rocco, D. J. Rosenberg, T. M. Ryan, S. Seifert, H. Sekiguchi, D. Svergun, S. Teixeira, A. Thureau, T. M. Weiss, A. E. Whitten, K. Wood and X. Zuo. A round-robin approach provides a detailed assessment of biomolecular small-angle scattering data reproducibility and yields consensus curves for benchmarking. 	Acta Crystallographica D78, 1315-1336 (2022).  DOI <a href="https://doi.org/10.1107/S2059798322009184" title="article DOI link"> 10.1107/S2059798322009184</a></li>
        <li> C. J. Lalaurie, A. Splevins, T. S. Barata, K. A. Bunting, D. R. Higazi, M. Zloh, Mire, V. A. Spiteri, S. J. Perkins, P. A. Dalby. Elucidation of critical pH-dependent structural changes in Botulinum Neurotoxin E. Journal of Structural Biology 214, 107876 (2022).  DOI <a href="https://doi.org/10.1016/j.jsb.2022.107876" title="article DOI link"> 10.1016/j.jsb.2022.107876</a></li>
        <li> A. Barclay, N. Tidemand Johansen, F. G. Tidemand, L. Arleth, M. C. Pedersen. Global fitting of multiple data frames from SEC-SAXS to investigate the structure of next-generation nanodiscs. Acta Cryst D 78, 483-493 (2022). DOI <a href="https://doi.org/10.1107/S2059798322001838" title="article DOI link"> 10.1107/S2059798322001838</a></li>
        <li> E. Brookes, M. Rocco. A database of calculated solution parameters for the recently released AlphaFold predicted protein structures. Sci Rep 12, 7349 (2022) DOI <a href="https://doi.org/10.1038/s41598-022-10607-z" title="article DOI link"> 10.1038/s41598-022-10607-z</a></li>
        <li> F. G. Tidemand, S. Blemmer, N. T. Johansen, L. Arleth, M. C. Pedersen. Non-ionic detergent assists formation of supercharged nanodiscs and insertion of membrane proteins. Biochimica et Biophysica Acta (BBA) - Biomembranes 1864, 183884 (2022). DOI <a href="https://doi.org/10.1016/j.bbamem.2022.183884" title="article DOI link"> 10.1016/j.bbamem.2022.183884</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35182589" title="pubmed link">35182589</a></li>
<br>

         <!-- 2021 -->
        <b>2021</b>
        <li> T. Cleveland, E. Blick, S. Krueger, A. Leung, T. Darwish and P. Butler. Direct Localization of Detergents and Bacteriorhodopsin in the Lipid Cubic Phase By Small-Angle Neutron Scattering. IUCRJ 8, 22-32 (2021). DOI <a href="https://doi.org/10.1107/S2052252520013974;" title="article DOI link"> 10.1107/S2052252520013974</a></li>
        <li> C. Jeong, R. Franklin, K.J. Edler, K. Vanommeslaeghe, S. Krueger and J.E. Curtis. Styrene-maleic Acid Copolymer Nanodiscs to Determine the Shape of Membrane Proteins, J. Phys. Chem B 126, 1034-1044 (2021). DOI <a href="https://doi.org/10.1021/acs.jpcb.1c05050 " title="article DOI link"> 10.1021/acs.jpcb.1c05050</a></li>
        <li> E, C. Arturo, G. W. Merkel, M. R. Hansen, S. Lisowski, D. Almeida, K. Gupta, E. K. Jaff. Manipulation of a cation-p sandwich reveals conformational flexibility in phenylalanine hydroxylase. Biochimie, Inborn errors of metabolism – biochemistry, structures and therapeutic design 183, 63-77 (2021). DOI <a href="https://doi.org/10.1016/j.biochi.2020.11.011" title="article DOI link"> 10.1016/j.biochi.2020.11.011</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33221376" title="pubmed link">33221376</a></li>
        <li> S. Donets, O. Guskova, J.-U. Sommer. Searching for Aquamelt Behavior among Silklike Biomimetics during Fibrillation under Flow. J. Phys. Chem. B 125, 3238-3250 (2021). DOI <a href="https://doi.org/10.1021/acs.jpcb.1c00647" title="article DOI link"> 10.1021/acs.jpcb.1c00647</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33750140" title="pubmed link">33750140</a></li>
        <li> F. Heinrich, Q. N. Van, F. Jean-Francois, A. G. Stephen, M. L&ouml;sche. Membrane-bound KRAS approximates an entropic ensemble of configurations. Biophysical Journal 120, 4055-4066 (2021). DOI <a href="https://doi.org/10.1016/j.bpj.2021.08.008" title="article DOI link"> 10.1016/j.bpj.2021.08.008</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34384763" title="pubmed link">34384763</a></li>
        <li> N. T. Johansen, M. Bonaccorsi, T. Bengtsen, A. H. Larsen, F. G. Tidemand, M. C. Pedersen, P. Huda, J. Berndtsson, T. Darwish, N. R. Yepuri, A. Martel, T. G. Pomorski, A. Bertarello, M. Sansom, M. Rapp, R. Crehuet, T. Schubeis, K. Lindorff-Larsen, G. Pintacuda, L. Arleth. Mg2+-dependent conformational equilibria in CorA and an integrated view on transport regulation. eLife 11, e71887 (2022). DOI <a href="https://doi.org/10.7554/eLife.71887" title="article DOI link"> 10.7554/eLife.71887</a></li>
        <li> A. H. Larsen, N. T. Johansen, M. Gajhede, L. Arleth, S. R. Midtgaard. Lipid-bound ApoE3 self-assemble into elliptical disc-shaped particles. Biochimica et Biophysica Acta (BBA) - Biomembranes 1863, 183495 (2021). DOI <a href="https://doi.org/10.1016/j.bbamem.2020.183495" title="article DOI link"> 10.1016/j.bbamem.2020.183495</a></li>
        <li> A. H. Larsen, M. C. Pedersen. Experimental noise in small-angle scattering can be assessed using the Bayesian indirect Fourier transformation. J Appl Cryst 54, 1281-1289 (2021). DOI <a href="https://doi.org/10.1107/S1600576721006877" title="article DOI link"> 10.1107/S1600576721006877</a></li>
        <li> J. Maco&scaron;ek, B. Simon, J.-B. Linse, P. K. A. Jagtap, S. L. Winter, J. Foot, K. Lapouge, K. Perez, M. Rettel, M. T. Ivanovic, P. Masiewicz, B. Murciano, M. M. Savitski, I. Loedige, J. S. Hub., F. Gabel, J. Hennig. Structure and dynamics of the quaternary hunchback mRNA translation repression complex. Nucleic Acids Research 49, 8866-8885 (2021). DOI <a href="https://doi.org/10.1093/nar/gkab635" title="article DOI link"> 10.1093/nar/gkab635</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34329466" title="pubmed link">34329466</a></li>
        <li> L. A. Perera, S. Preissler, N. R. Zaccai, S. Pr&eacute;vost, J. M. Devos, M. Haertlein, D. Ron. Structures of a deAMPylation complex rationalise the switch between antagonistic catalytic activities of FICD. Nat Commun 12, 5004 (2021).  DOI <a href="https://doi.org/10.1038/s41467-021-25076-7" title="article DOI link"> 10.1038/s41467-021-25076-7</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34408154" title="pubmed link">34408154</a></li>
        <li> A. S. Solovyova, D. T. Peters, G. Dura, H. Waller, J. H. Lakey, D. A. Fulton. Probing the oligomeric re-assembling of bacterial fimbriae in vitro: a small-angle X-ray scattering and analytical ultracentrifugation study. Eur Biophys J 50, 597-611 (2021). DOI <a href="https://doi.org/10.1007/s00249-021-01543-3" title="article DOI link"> 10.1007/s00249-021-01543-3</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33948690" title="pubmed link">33948690</a></li>
		<li> J. Sonje, S. Thakral, S. Krueger, R. Suryanarayanan. Reversible Self-association in lactate dehydrogenase during Freeze-thaw in Buffered Solutions using Neutron Scattering. Mol. Pharm., 18, 4459-4474. (2021). DOI <a href="https://doi.org/10.1021/acs.molpharmaceut.1c00666" title="article DOI link">10.1021/acs.molpharmaceut.1c00666</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34709831" title="pubmed link">34709831</a></li>
		<li>P. Calyam, N. Wilkins-Diehr, M. Miller, E. H. Brookes, R. Arora, A. Chourasia, D. M. Jennewein, V. Nandiga, M. Drew LaMar, S. B. Cleveland, G. Newman, S. Wang, I. Zaslavsky, M. A. Cianfrocco, K. Ellett, D. Tarboton, K. G. Jeffery, Z. Zhao, J. Gonzalez-Aranda, M. J. Perri, G. Tuckers, L. Candela, T. Kiss, S Gesing. Measuring success for a future vision: Defining impact in science gateways/virtual research environments. Concurrency Computation Practrice and Experience 33,e6099 (2021). DOI <a href="https://doi.org/10.1002/cpe.6099" title="article DOI link">10.1002/cpe.6099</a></li>
		<li>V. A. Spiteri, M. Goodall, J. Doutch, R. P. Rambo, J. Gor, S. J. Perkins. Solution structures of human myeloma IgG3 antibody reveal extended Fab and Fc regions relative to the other IgG subclasses. J. Biol. Chem. 297, 100995 (2021). DOI <a href="https://doi.org/10.1016/j.jbc.2021.100995" title="article DOI link">10.1016/j.jbc.2021.100995</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34302810" title="pubmed link">34302810</a></li>
		<li>V. A. Spiteri, J. Doutch, R. P. Rambo, J. Gor, P. A. Dalby, S. J. Perkins. Solution structure of deglycosylated human IgG1 shows the role of CH2 glycans in its conformation. Biophysical J. 200, 1814-1834 (2021). DOI <a href="https://doi.org/10.1016/j.bpj.2021.02.038" title="article DOI link">10.1016/j.bpj.2021.02.038</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33675758" title="pubmed link">33675758</a></li>
		<li>C.R. Horne, H. Venugopal, S. Panjikar, D.M. Wood, A. Henrickson, E. Brookes, R.A. North, J.M. Murphy, R. Friemann, M.D. Griffin, G. Ramm. Mechanism of NanR gene repression and allosteric induction of bacterial sialic acid metabolism. Nature communications, 12(1), 1-16 (2021). DOI <a href="https://doi.org/10.1038/s41467-021-22253-6" title="article DOI link">10.1038/s41467-021-22253-6</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33790291" title="pubmed link">33790291</a></li>
        <li>O. M. Dunne, X. Gao, R. Nan, J. Gor, P. J. Adamson, D. L. Gordon, M. Moulin, M. Haertlein, T. V. Forsyth, S. J. Perkins. A Dimerization Site at SCR-17/18 in Factor H Clarifies a New Mechanism for Complement Regulatory Control. Frontiers in Immunology 11, 3536 (2021). DOI <a href="https://doi.org/10.3389/fimmu.2020.601895" title="article DOI link">10.3389/fimmu.2020.601895</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33552059" title="pubmed link">33552059</a></li>
		<li>S. Krueger, J. E. Curtis, D. R. Scott, A. Grishaev, G. Glenn, G. Smith, L. Ellingsworth, O. Borisov, and E. L. Maynard.  Structural Characterization and Modeling of a Respiratory Syncytial Virus Fusion Glycoprotein. Molecular Pharmaceutics 18 (1), 359-376 (2021). DOI <a href="https://doi.org/10.1021/acs.molpharmaceut.0c00986" title="article DOI link">10.1021/acs.molpharmaceut.0c00986</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33322901" title="pubmed link">33322901</a></li>
<br>

         <!-- 2020 -->
        <b>2020</b>
        <li>M.Stoutjesdyk, E. Brookes, A. Henrickson, B. Demeler. Measuring compressibility in the optima AUC analytical ultracentrifuge. European Biophysics Journal, 49(8), 711-718 (2020). DOI <a href="https://doi.org/10.1007/s00249-020-01482-5" title="article DOI link">10.1007/s00249-020-01482-5</a>.</li>
		<li>N. Kadkhodayi-Kholghi, J. S. Bhatt, J. Gor, L. C. McDermott, D. P. Gale, J. S. Perkins. The solution structure of the human complement regulator FHR5 reveals a compact dimer and provides new insights on FHR5 nephropathy. J. Biol. Chem. 295, 16342-16358 (2020). DOI: <a href="https://doi.org/10.1074/jbc.RA120.015132" title="article DOI link">10.1074/jbc.RA120.015132</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32928961" title="pubmed link"> 32928961</a></li>
		<li>R. Saxena, C. B. Stanley, P. Kumar, M. J. Cuneo, D. Patil, J. Jha, K. L. Weiss, D. K. Chattoraj, E. Crooke. A nucleotide-dependent oligomerization of the Escherichia coli replication initiator DnaA requires residue His136 for remodeling of the chromosomal origin. Nucleic Acids Research 48, 200-211 (2020) DOI: <a href="https://doi.org/10.1093/nar/gkz939" title="artilce DOI link">10.1093/nar/gkz939</a></li>
		<li>D. C. Marx, A. M. Plummer, A. M. Faustino, T. Devlin, M. A. Roskopf, M. J. Leblanc, H. J. Lessen, B. T. Amann, P. J. Fleming, S. Krueger, S. D. Fried, K. G. Fleming. SurA is a cryptically grooved chaperone that expands unfolded outer membrane proteins. PNAS 117, 28026-2835 (2020). DOI <a href="https://doi.org/10.1073/pnas.2008175117" title="article DOI link">10.1073/pnas.2008175117</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33093201" title="pubmed link">33093201</a></li>
		<li>D. C. Marx, M.J. Leblanc, A. M. Plummer, S. Krueger, K. G. Fleming. Domain interactions determine the conformational ensemble of the periplasmic chaperone SurA. Protein Science 29, 2043–2053 (2020). DOI <a href="https://doi.org/10.1002/pro.3924" title="article DOI link">10.1002/pro.3924</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32748422" title="pubmed link">32748422</a></li>
		<li>T. A. Ronnebaum, K. Gupta, D. W. Christianson. Higher-order oligomerization of a chimeric &alpha;&beta;&gamma; bifunctional diterpene synthase with prenyltransferase and class II cyclase activities is concentration-dependent. J. Struct. Biol. 210, 107463 (2020).  DOI: <a href="https://doi.org/10.1016/j.jsb.2020.107463" title="artilce DOI link">10.1016/j.jsb.2020.107463</a></li>
		<li>A. J. Boughton, S. Krueger, D. Fushman. Branching via K11 and K48 Bestows Ubiquitin Chains with a Unique Interdomain Interface and Enhanced Affinity for Proteasomal Subunit Rpn1. Structure 28, 29-43.e6 (2020). DOI: <a href="https://doi.org/10.1016/j.str.2019.10.008" title="artilce DOI link">10.1016/j.str.2019.10.008</a>;  Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3167789" title="pubmed link">31677891</a></li>
<br>
	
         <!-- 2019 -->
        <b>2019</b>
		<li>L. Puster, C. Stanley, V. Uversky, J. Curtis, S. Krueger, Y. Chu, C. Peterson. Characterization of an Extensive Interface on Vitronectin for Binding to Plasminogen Activator Inhibitor-1: Adoption of Structure in an Intrinsically Disordered Region. Biochemistry, 58, 5117-5134 (2019). DOI: <a href="https://doi.org/10.1021/acs.biochem.9b00605">10.1021/acs.biochem.9b00605</a>;  Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31793295" title="pubmed link">31793295</a></li>
		<li>M. Bush , B. M. Alhanshali, S. Qian, C. B. Stanley , W. T. Heller , T. Matsui , T. M. Weiss , I. D. Nicholl, T. Walz, D. J. E. Callaway, Z. Bu. An ensemble of flexible conformations underlies mechanotransduction by the cadherin-catenin adhesion complex. PNAS 116, 21545-21555 (2019). DOI: <a href="https://doi.org/10.1073/pnas.1911489116" title="artilce DOI link">10.1073/pnas.1911489116</a></li>
		<li>D. W. Wright, E. L. K. Elliston, G. K. Hui, S. J. Perkins. Atomistic modelling of scattering curves for human IgG1/4 reveals new structure-function insights. Biophysical Journal 117, 2101-2119 (2019) DOI: <a href="https://doi.org/10.1016/j.bpj.2019.10.024" title="artilce DOI link">10.1016/j.bpj.2019.10.024</a></li>
		<li>A. Y. Xu ,M. M. Castellanos, K. Mattison, S. Krueger, J. E. Curtis. Studying Excipient Modulated Physical Stability and Viscosity of Monoclonal Antibody Formulations Using Small-Angle Scattering. Mol. Pharmaceutics 16, 4319-4338 (2019) DOI: <a href="https://pubs.acs.org/doi/10.1021/acs.molpharmaceut.9b00687" title="artilce DOI link">10.1021/acs.molpharmaceut.9b00687</a></li>
		<li>E. Brookes, J. Stubbs. GenApp, Containers and Abaco. PEARC19 - Proceedings of the Practice and Experience in Advanced Research Computing on Rise of the Machines (learning) Article No. 12. (2019) DOI: <a href="https://doi.org/10.1145/3332186.3332191">10.1145/3332186.3332191</a></li>
		<li>S. Bowerman, J. E. Curtis, J. Clayton, E. H. Brookes, J. Wereszczynski. BEES: Bayesian Ensemble Estimation from SAS. Biophysical Journal 117, 399-407 (2019) DOI: <a href="https://doi.org/10.1016/j.bpj.2019.06.024" title="artilce DOI link">10.1016/j.bpj.2019.06.024</a></li>
        <li>A. McCluskey, J. Grant, A. Smith, J. Rawle, D. J. Barlow, M. J. Lawrence, S. Parker, K. J. Edler. Assessing molecular simulation for the analysis of lipid monolayer reflectometry. Journal of Physics Communications 3, 075001 (2019)  DOI: <a href="https://iopscience.iop.org/article/10.1088/2399-6528/ab12a9" title="artilce DOI link">10.1088/2399-6528/ab12a9</a> (Open Access)</li>
		<li>G.K. Hui, A.D. Gardener, H. Begum, J. Gor, S.J. Perkins. The solution structure of the human IgG2 subclass is distinct from those for human IgG1 and IgG4 providing an explanation for their discrete functions. J. Biol. Chem. 294, 10789-10806 (2019) DOI: <a href="https://doi.org/10.1074/jbc.RA118.007134" title="artilce DOI link">10.1074/jbc.RA118.007134</a>;  Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31088911" title="pubmed link">31088911</a></li>
		<li>A. Savelyev, E. Brookes. GenApp: Extensible tool for rapid generation of web and native GUI applications. Future Generation Computer Systems 94, 929-93 (2019) DOI: <a href="https://doi.org/10.1016/j.future.2017.09.069" title="artilce DOI link">10.1016/j.future.2017.09.069</a></li>
        <li>A.R. McCluskey, J. Grant, A.R. Symington, T. Snow, J. Doutch, B.J. Morgan, S.C. Parker, K.J. Edler. An introduction to classical molecular dynamics simulation for experimental scattering users. J. Appl. Cryst. 52, 665-668 (2019) DOI: <a href="https://doi.org/10.1107/S1600576719004333" title="artilce DOI link">10.1107/S1600576719004333</a> (Open Access)</li>
		<li>N. Codina, D. Hilton, C. Zhang, N. Chakroun, S.S. Ahmad, S.J. Perkins* & P. A. Dalby*. An expanded conformation of an antibody Fab region by X-ray scattering, molecular dynamics and smFRET identifies an aggregation mechanism. J. Mol. Biol. 41, 1409-1425 (2019) [*joint corresponding authors] DOI: <a href="https://doi.org/10.1016/j.jmb.2019.02.009" title="article DOI link">10.1016/j.jmb.2019.02.009</a>; Pubmed: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30776431" title="pubmed link">30776431</a></li>
		<li>A.R. McCluskey, ; A. Sanchez-Fernandez,; K.J. Edler; S. Parker; A.J. Jackson; R.A. Campbell; T. Arnold. Bayesian determination of the effect of a deep eutectic solvent on the structure of lipid monolayers. Phys. Chem., Chem. Phys. 21, 6133-6141 (2019) DOI: <a href="https://doi.org/10.1039/C9CP00203K" title="article DOI link">10.1039/C9CP00203K</a>. <a href="https://arxiv.org/abs/1810.07616" title="link to archive">(arXiv:1810.07616 [cond-mat.soft]</a> - Open Acess)</li>
<br>

        <!-- 2018 -->
        <b>2018</b>
        <li>A.J. Osborne, R. Nan, A. Miller, J.S. Bhatt, J. Gor, S.J. Perkins. Two distinct conformations of factor H regulate discrete complement-binding functions in the fluid phase and at cell surfaces. J. Biol. Chem. 293, 17166-17187 (2018) DOI: <a href="https://doi.org/10.1074/jbc.RA118.004767" title="article DOI link">10.1074/jbc.RA118.004767</a>; Pubmed:  <a href="https://www.ncbi.nlm.nih.gov/pubmed/30217822" title="pubmed link">30217822</a> (Open Access)</li>
        <li>A. R. McCluskey, B. J. Morgan, K. J. Edler, S. C. Parker. pylj: A teaching tool for classical atomistic simulation. J. Open Source Educ. 1, 19 (2018) DOI: <a href="https://doi.org/10.21105/jose.00019" title="artilce DOI link">10.21105/jose.00019</a> (Open Access)</li>
		<li>E. Brookes, M. Rocco. Recent advances in UltraScan SOlution MOdeller (US-SOMO) hydrodynamic and small-angle scattering data analysis simulation suite. Eur. Biophys. J. https://doi.org/10.1007/s00249-018-1296-0 (2018)</li>
		<li>M.M. Castellanos, K. Mattison, S. Krueger, J.E. Curtis. Characterization of the NISTmAb Reference Material using small-angle scattering and molecular simulation. Part II: Concentrated protein solutions. Anal. Bioanal. Chem. 410, 2161-2171 (2018).</li>
		<li>M.M. Castellanos, S.C. Howell, D.T. Gallagher, J.E. Curtis. Characterization of the NISTmAb Reference Material using small-angle scattering and molecular simulation. Part I: Dilute protein solutions. Anal. Bioanal. Chem. 410, 2141-2159 (2018).</li>
<br>
		
	    <!-- 2017 -->
        <b>2017</b>
		<li>W. Zhang, S.C. Howell, D.W. Wright, A. Heindel, X. Qiu, J. Chen, J.E. Curtis. Combined Monte Carlo / Torsion-Angle Molecular Dynamics for Ensemble Modeling of Proteins, Nucleic Acids, and Carbohydrates. J. Mol. Graph. Model. 73, 179-190 (2017).</li>
		<li>M. M Castellanos, J.A. Snyder, M. Lee, S. Chakravarthy, N.J. Clark, A. McAuley, J.E. Curtis. Characterization of Monoclonal Antibody?Protein Antigen Complexes Using Small-Angle Scattering and Molecular Modeling. Antibodies. 6(4), 25, (2017).</li>
		<li>P. P. Bhojane, M. R. Duff, Jr., K. Bafna, P. Agarwal, C. Stanley, E. E. Howell. Small Angle Neutron Scattering Studies of R67 Dihydrofolate Reductase, a Tetrameric Protein with Intrinsically Disordered N-Termini. Biochemistry 56 (44), 5886-5899 (2017) DOI: <a href="https://doi.org/10.1021/acs.biochem.7b00822" title="article DOI link">10.1021/acs.biochem.7b00822</a></li>
		<li>K.T. Walker, R. Nan, D.W. Wright, J. Gor, A.C. Bishop, G.I. Makhatadze, B. Brodsky, S.J. Perkins. Non-linearity of the collagen triple helix in solution and implication for collagen function. Biochem. J., 474, 2203-2217, (2017).</li>
		<li>R. Nan, C.M. Furze, D.W. Wright, J. Gor, R. Wallis, S.J. Perkins. Flexibility in mannan-binding lectin-associated serine proteases-1 and -2 provides novel insight on lectin pathway activation. Structure, 25, 364-375, (2017). <a href="http://www.sciencedirect.com/science/article/pii/S096921261630404X">Interactive model viewer</a></li>
		<li>M.M. Castellanos, A. McAuley, J.E. Curtis. Investigating Structure and Dynamics of Proteins in Amorphous Phases Using Neutron Scattering. Comp. Struct. Biotech. J.,  15, 117-130, (2017).</li>
<br>

        <!-- 2016 -->
        <b>2016</b>
		<li>Z. Jia, S.K. Whitaker, J.M. Tomich, J. Chen. Organization and structure of branched oligopeptide bilayers. Langmuir. 32 9883-9891 (2016).</li>
        <li>K.-W. Fung, D.W. Wright, J. Gor, M.J. Swann, S.J. Perkins. Domain structure of human complement C4b extends with increasing NaCl concentrations: implications for its regulatory mechanism. Biochem. J. 473, 4473-4491 (2016).</li>
		<li>E.H. Brookes, P. Vachette, J. Perez, M. Rocco. US-SOMO HPLC-SAXS module: dealing with capillary fouling and extraction of pure component patterns from poorly resolved SEC-SAXS data. J. Appl. Cryst. 49, 1827-1841, (2016).</li>
		<li>M.M. Castellanos, N.J. Clark, M.C. Watson, S. Krueger, A. McAuley, J.E. Curtis. Role of Molecular Flexibility and Colloidal Descriptions of Proteins in Crowded Environments from Small-Angle Scattering. J. Phys. Chem. B., 120(49), 12511-12518, (2016).</li>
		<li><a href="http://journals.iucr.org/j/issues/2016/06/00/zg5003/index.html">S. J. Perkins, D. W. Wright, H. Zhang, E. H. Brookes, J. Chen, T. C. Irving, S. Krueger, D. J. Barlow, K. J. Edler, D. J. Scott, N. J. Terrill, S. M. King, P. D. Butler, J. E. Curtis. Atomistic modelling of scattering data in the Collaborative Computational Project for Small Angle Scattering (CCP-SAS). J. Appl. Cryst. 49(6), 1861-1875, (2016)</a>.</li>
        <li>T. Eisemann, M. McCauley, M.-F. Langelier, K. Gupta, S. Roy, G.D. Van Duyne, J.M. Pascal. Tankyrase-1 Ankyrin Repeats Form an Adaptable Binding Platform for Targets of ADP-Ribose Modification. Structure. 24, 1679?1692, (2016).</li>
        <li>S.C. Howell, X. Qiu, J.E. Curtis. Monte Carlo simulation algorithm for B-DNA. J. Comp. Chem. 37(29), 2553-2563 (2016). <a href="http://onlinelibrary.wiley.com/doi/10.1002/jcc.24510/full">Journal cover image</a></li>
        <li>Z. Jia, J. Chen. Necessity of High-Resolution for Coarse-Grained Modeling of Flexible Proteins. J. Comp. Chem., 37(18), 1725-1733, (2016).</li>
		<li>M. Green, L. Hatter, E. Brookes, P. Soultanas, D. J. Scott. Defining the Intrinsically Disordered C-Terminal Domain of SSB Reveals DNA-Mediated Compaction. J. Mol. Biol., 428(2), 357?364, (2016).</li>
		<li>H. Zahid, L. Miah, A.M. Lau, L. Brochard, D. Hati, T.T.T Bui, A.F. Drake, J. Gor, S.J. Perkins and L.C. McDermott. Zinc-induced oligomerisation of zinc a2 glycoprotein reveals multiple fatty acid binding sites. Biochem. J. 473, 43-54 (2016).</li>
		<li>C. A. Castaneda, A. Chaturvedi, C. M. Camara, J. E. Curtis, S. Krueger, D. Fushman. Linkage-specific conformational ensembles of non-canonical polyubiquitin chains. Phys. Chem. Chem. Phys., 18, 5771-5788 (2016). <a href="http://www.sciencedirect.com/science/article/pii/S0969212616000915">Structure Preview article</a></li>
		<li>C. A. Castaneda, E. Dixon, O. Walker, A. Chaturvedi, M. A. Nakasone, J. E. Curtis, M. R. Reed, S. Krueger, T. A. Cropp, D. Fushman. Linkage via k27 bestows ubiquitin chains with unique properties among polyubiquitins. Structure 24, 423-436 (2015).</li>
        <li>K. H. Lee, J. Chen. Multiscale Enhanced Sampling of Intrinsically Disordered Protein Conformations. Journal of Computational Chemistry. 37, 550-557 (2016).</li>
        <li>E. H. Brookes, A. Kapoor, P. Patra, S. Marru, R. Singh, M. Pierce. GSoC 2015 Student contributions to GenApp and Airavata. Concurrency and Computation: Practice and Experience. 28(7), 1960, (2016).</li>
<br>

        <!-- 2015 -->
        <b>2015</b>
		<li>N. R. Zaccai, C. W. Sandlin, J. T. Hoopes, J. E. Curtis, P. J. Fleming, K. G. Fleming, S. Krueger. Deuterium Labeling Together with Contrast Variation Small-Angle Neutron Scattering Suggests How Skp Captures and Releases Unfolded Outer Membrane Proteins. Methods in Enzymology. 566, 159-210, (2015).</li>
		<li>G. K. Hui, D. W. Wright, O. L. Vennard, L. E. Rayner, M. Pang, S. C. Yeo, J. Gor, K. Molyneux, J. Barratt & S. J. Perkins. The solution structures of native and patient monomeric human IgA1 reveal asymmetric extended structures: implications for function and IgAN disease. Biochem. J. 471, 167-185 (2015).</li>
		<li>B. J. Tarasevich, J. S. Philo, N. K. Maluf, S. Krueger, G. W. Buchko, G. Lin, W. J. Shaw. The leucine-rich amelogenin protein (LRAP) is primarily monomeric and unstructured in physiological solution. J. Struct. Biol. 190, 81?91 (2015).</li>
		<li>E. H. Brookes, N. Anjum, J. E. Curtis, S. Marru, R. Singh, M. Pierce. The GenApp framework integrated with Airavata for managed compute resource submissions. Concurrency and Computation: Practice and Experience. 27(16), 4292, (2015).</li>
		<li>D. W. Wright, R. Nan, G.-K. Hui, J. E. Curtis, E. H. Brookes, S. J. Perkins. CCP-SAS - Novel Approaches for the Atomistic Modelling of Small Angle Scattering Data in Biology. Biophysical Journal. 108(2), 191a (2015).</li>
		<li>L. E. Rayner, G. K. Hui, J. Gor, R. K. Heenan, P. A. Dalby, S. J. Perkins. The solution structures of two human IgG1 antibodies show conformational stability and accommodate their C1q and Fc?R ligands. J. Biol. Chem. 290, 8420-8438 (2015).</li>
		<li>Zhao, H., Ghirlando, R., Alfonso, C., Arisaka, F., Attali, I., Bain, D. L., Bakhtina, M. M., Becker, D. F., Bedwell, G. J., Bekdemir, A., Besong, T. M. D., Birck, C., Brautigam, C. A., Brennerman, W., Byron, O., Bzowska, A., Chaires, J. B., Chaton, C. T., Cölfen, H., Connaghan, K. D., Crowley, K. A., Curth, U., Daviter, T., Dean, W. L., Díez, A. I., Ebel, C., Eckert, D. M., Eisele, L. E., Eisenstein, E., England, P., Escalante, C., Fagan, J. A., Fairman, R.,  Finn, R. M., Fischle, W., García de la Torre, J., Gor, J., Gustafsson, H., Hall, D., Harding, S. E., Hernández-Cifre, J. G., Herr, A. B., Howell, E. E., Isaac, R. S., Jao, S.-C., Jose, D., Kim, S.-J., Kokona, B., Kornblatt, J. A., Kosek, D., Krayukhina, E., Krzizike, D., Kusznir, E. A., Kwon, H., Larson, A., Laue, T. M., Le Roy, A., Leech, A. P., Lilie, H., Luger, K., Luque-Ortega, J. R., Ma, J., May, C. A., Maynard, E. L., Modrak-Wojcik, A., Mok, Y.-F., Mücke, N., Nagel-Steger, L., Narlikar, G. J., Noda, M., Nourse, A., Obsil, T., Park, C. K., Park, J.-K., Pawelek, P. D., Perdue, E. E., Perkins, S. J., Perugini, M. A., Peterson, C. L., Peverelli, M. G., Piszczek, G., Prag, G., Prevelige, P. E., Raynal, B. D. E., Rezabkova, L., Richter, K., Ringel, A. E., Rosenberg, R., Rowe, A. J., Rufer, A. C., Scott, D. J., Seravalli, J. G., Solovyova, A. S., Song, R., Staunton, D., Stoddard, C., Stott, K.,  Strauss, H. M., Streicher, W. W., Sumida, J. P., Swygert, S. G., Szczepanowski, R. H., Tessmer, I., Toth, IV, R. T., Tripathy, A., Uchiyama, S., Uebel, S. F. W., Unzai, S., Vitlin-Gruber, A., von Hippel, P. H., Wandrey, C., Wang, S.-H.,  Weitzel, S. E., Wielgus-Kutrowska, B., Wolberger, C., Wolff, M., Wright, E., Wu, Y.-S., Wubben, J. M. & Schuck, P. A multilaboratory comparison of calibration accuracy and the performance of external references in analytical ultracentrifugation. PLOS One. 10(5):e0126420 (2015).</li>
		<li>D. W. Wright, S.J. Perkins, (2015). SCT: A suite of programs for comparing atomistic models to small angle scattering data. J. Appl. Crystallographr. 48(3), 953-961 (2015).</li>
		<li>E. Rodriguez, R. Nan, K. Li, J. Gor, S. J. Perkins. A revised mechanism for the activation of complement C3 to C3b: a molecular explanation of a disease-associated polymorphism. J. Biol. Chem. 290, 2334-2350 (2015).</li>
<br>

        <!-- 2014 -->
        <b>2014</b>
		<li>W. Zhang, J. Chen. Replica exchange with guided annealing for accelerated sampling of disordered protein conformations. Journal of Computational Chemistry. 35(23), 1682 (2014).</li>
		<li>Y. Peng, J.E. Curtis, X. Fang, S.A. Woodson. Structural model of an mRNA in complex with the bacterial chaperone Hfq. Proc. Nat. Acad. Sci. 111 (48), 17134-17139 (2014).</li>
		<li>S. Krueger, J. H. Shin, J. E. Curtis, K. A. Rubinson, Z. Kelman. The solution structure of full-length dodecameric MCM by SANS and molecular modeling. Proteins: Structure, Function, and Bioinformatics 82, 2364-2374 (2014).</li>
		<li>J.E. Curtis, H. Zhang, H. Nanda. SLDMOL: A tool for the structural characterization of thermally disordered membrane proteins. Comp. Phys. Comm. 185, 3010-3015 (2014). <a href="http://www.sciencedirect.com/science/article/pii/S0010465514002483">Software description</a></li>
		<li>Z. Zhang, J.-M.Y. Carrillo, S.-K. Ahn, B. Wu, K. Hong, G.S. Smith, C. Do. Atomistic Structure of Bottlebrush Polymers: Simulations and Neutron Scattering Studies. Macromolecules 47, 5808 (2014).</li>
		<li>L.E. Rayner, G.K. Hui, J. Gor, R.K. Heenan, P.A. Dalby and S.J. Perkins. The Fab Conformations in the Solution Structure of Human Igg4 restricts Access to its Fc region: Implications for Functional Activity. J. Biol. Chem. 289, 20740-20756 (2014).</li>
		<li>N.J. Clark, M. Raththagala, N.T. Wright, E.A. Buenger, J.F. Schildbach, S. Krueger, J.E. Curtis. Structures of TraI in solution. J. Mol. Model. 20, 2308 (2014).</li>
		<li>M.C. Watson and J.E. Curtis. Probing the average local structure of biomolecules using small-angle scattering and scaling laws. Biophys. J. 106 (11), 2474-2482 (2014).</li>
		<li>E.J. Yearly, P.D. Godfrin, T. Perevozchikova, H. Zhang, P. Falus, L. Porcar, M. Nagao, J. E.Curtis, P. Gawande, R. Taing, I.E. Zarraga, N.J. Wagner, Y. Liu. Observation of small cluster formation in concentrated monoclonal antibody solutions and its implications to solution viscosity. Biophys. J. 106, 1763-1770 (2014). <a href="http://pubs.acs.org/iapps/wld/cen/results.html?line3=monoclonal-antibody">Highlighted in C&E News.</a></li>
		<li>S.J. Perkins, K.-W. Fung, S. Khan. Molecular interactions between complement factor H and its heparin and heparan sulphate ligands. Frontiers in Immunology. 5, 126, 1-14 (2014).</li>
<br>

        <!-- 2013 -->
        <b>2013</b>
		<li>L.E. Rayner, N. Kadkhodayi-Kholghi, R.K. Heenan, J. Gor, P.A. Dalby, S.J. Perkins. The solution structure of rabbit IgG accounts for its interactions with the Fc receptor and complement C1q and its conformational stability. J. Mol. Biol. 425, 506-523 (2013).</li>
		<li>S. Khan, K.-W. Fung, E. Rodriguez, R. Patel, J. Gor, B. Mulloy, S. J. Perkins. The solution structure of heparan sulphate differs from that of heparin: implications for function. J. Biol. Chem. 288, 27737-27751 (2013).</li>
		<li>E. Brookes, J. P&#233;rez, B. Cardinali, A. Profumo, P. Vachette and M. Rocco. Fibrinogen species as resolved by HPLC-SAXS data processing within the UltraScan Solution Modeler (US-SOMO) enhanced SAS module. J Appl Cryst. 46 , 1823(2013).</li>
		<li>N.J. Clark, H. Zhang, S. Krueger, H. J. Lee, R.R. Ketchem, B.A. Kerwin, S.R. Kanapuram, M.J. Treuheit, A. McAuley, J.E. Curtis. Small-angle neutron scattering study of a monoclonal antibody using free-energy constraints. J. Phys. Chem. B 117, 14029-14038 (2013).</li>
		<li>K.L. Sarachan, J.E. Curtis, S. Krueger. Small-angle scattering contrast calculator for protein and nucleic acid complexes in solution. J. Appl. Cryst. 46 1889-1893 (2013).</li>
		<li>H. Zhang, S. Khodadadi, S.L. Fiedler, J. E. Curtis. Role of water and ions on the dynamical transition of RNA. J. Phys. Chem. Letters 4, 3325-3329 (2013).</li>
		<li>M.C. Watson and J.E. Curtis. Rapid and accurate calculation of small-angle scattering profiles using the golden ratio. J. Appl. Cryst. 46, 1171-1177 (2013).</li>
	</ul>
   </div>

   <h3>Presentations given on CCP-SAS software and its applications:</h3>

   <div id="linkList">
	<ul>
        <!-- 2019 -->
        <b>2019</b>
        <li> Y. Chen, C. Jeong, A. Savelyev, S.nKrueger, J. E. Curtis, E. H. Brookes, D. Fushman. ROTDIF-web and ALTENS: GenApp-based Science Gateways for Biomolecular Nuclear Magnetic Resonance (NMR) Data Analysis and Structure Modeling. Gateways 2019. San Diego, California. September 23-25, 2019. DOI: <a href="https://doi.org/10.17605/OSF.IO/T4GKH" title="artilce DOI link">10.17605/OSF.IO/T4GKH</a></li>
		<li>M. Christie, M. Owczarzak, Y-S. Ho, P. Parsons, S. Krueger, J. E. Curtis, E. H. Brookes. GenApp Style and Layout Enhancements. Gateways 2019. San Diego, California. September 23-25, 2019. DOI: <a href="https://doi.org/10.17605/OSF.IO/NQSPR" title="artilce DOI link">10.17605/OSF.IO/NQSPR</a>. <b>Poster</b></li>
		<li>A. Chourasia, D. Nedau, J. Luo, T. Chen, M. Miller, E. Brookes. Enabling rich data sharing for Science Gateways via the SeedMeLab platform. Gateways 2019. San Diego, California. September 23-25, 2019. DOI: <a href="https://doi.org/10.17605/OSF.IO/WV3BF" title="artilce DOI link">10.17605/OSF.IO/WV3BF</a></li>
		<li>N. Wilkins-Diehr, M. Miller, E. H. Brookes, R. Arora, A. Chourasia, P. Calyam, D. M. Jennewein, V. Nandigam, M. D. LaMar, S. B. Cleveland, G. Newman, S. Wang, I. Zaslavsky, M. A. Cianfrocco, K. Ellett, D. Tarboton. Measuring Success: How Science Gateways Define Impact. Gateways 2019. San Diego, California. September 23-25, 2019.</li>
		<li>C. Jeong, P. Vachette, E. H. Brookes. MultiSAXSHub: A GenApp-Generated Science Gateway for Unified Computational SAXS Modeling Tool. Gateways 2019. San Diego, California. September 23-25, 2019. DOI: <a href="https://doi.org/10.17605/OSF.IO/354CJ17" title="artilce DOI link">10.17605/OSF.IO/354CJ17</a>. <b>Poster</b></li>
		<li>A Guan, M. Casertano, C. Jeong, E. H. Brookes, D. Fushman. A GenApp-Generated Science Gateway for Comprehensive Analysis of Paramagnetic Effects in Biomolecular NMR Data. Gateways 2019. San Diego, California. September 23-25, 2019. DOI: <a href="https://doi.org/10.17605/OSF.IO/92VCY" title="artilce DOI link">10.17605/OSF.IO/92VCY</a>.</li>
        <li>G. K. Hui, A. D. Gardener, H. Begum, J. Gor, S. J. Perkins. The solution structure of the human IgG2 subclass is distinct from those for human IgG1 and IgG4, providing an explanation for their discrete functions. 24th International Analytical Ultracentrifugation Workshop and Symposium. Christchurch, New Zealand, August 24-29, 2019.</li>
		<li>V. A. Spiteri, C. F. S. Eldrid, J. Doutch, R. P. Rambo, J. Gor, K. Thalassinos, P. A. Dalby, S. J. Perkins. Unravelling the solution structures and stabilities of therapeutic antibodies with and without glycans. 24th International Analytical Ultracentrifugation Workshop and Symposium. Christchurch, New Zealand, August 24-29, 2019.</li>
		<li>E. Brookes, J. Stubbs. GenApp, Containers and Abaco. PEARC19 Practice and Experience in Advanced Research Computing: RISE OF THE MACHINES [learning]. Chicago, Ill. July 29-August 1, 2019.</li>
		<li>K. Edler - Micelles & Monolayers: Exploring use of models to understand scattering data. CECAM - Simulations of Small angle scattering for soft matter and life sciences. Institut-Laue-Langevin, Grenoble, France. 8-12 April 2019</li>
        <li>S. Perkins, J. Bhatt --- CCP-SAS - a community consortium for the advanced modelling of scattering data - applications and implementation. CECAM - Simulations of Small angle scattering for soft matter and life sciences. Institut-Laue-Langevin, Grenoble, France. 8-12 April 2019</li>
        <li>J. Clayton, J. Wereszczynski. Using goal-oriented enhanced sampling to understand SAXS and NMR data of IsdH from Staphylococcus aureus. 63rd Annual Meeting of the Biophysical Society, March 2-6, 2019, Baltimore Maryland, USA. <b>Poster</b></li>
        <li>J. Wereszczynski - Exploring the Relationship Between the Structure, Function, and Dynamics of Biomolecular Complexes - University of Delaware Department of Physics, February 2019</li>
        <li>J. Wereszczynski - Exploring the Relationship Between the Structure, Function, and Dynamics of Biomolecular Complexes - West Virginia University Department of Chemistry, January 2019</li>

        <!-- 2018 -->
        <b>2018</b>
        <li>E.H. Brookes, J. P&eacute;rez, P. Vachette and M. Rocco. Advances to the Ultrascan Solution Modeler (US-SOMO) SAS and HPLC-SAXS modules. XVII International Small Angle Scattering Conference - SAS2018 October 7-12, 2018, Traverse City, Michigan USA.</li>
        <li>E.H. Brookes, J.E. Curtis, D. Fushman, S. Krueger, A. Savelyev. The GenApp Framework for SAS and Beyond. XVII International Small Angle Scattering Conference - SAS2018 October 7-12, 2018, Traverse City, Michigan USA. <b>Poster</b></li>
        <li> E.H. Brookes. PVP - P-Value Pair Map. XVII International Small Angle Scattering Conference - SAS2018 October 7-12, 2018, Traverse City, Michigan USA. <b>Poster</b></li>
        <li>J.E. Curtis, C. Jeonga, R. Franklin, K.J. Edler. Towards truly stealth nanodiscs. XVII International Small Angle Scattering Conference - SAS2018 October 7-12, 2018, Traverse City, Michigan USA. <b>Poster</b></li>
        <li>A.Y. Xu, M.M. Castellanos, K. Mattison, S. Krueger, J.E. Curtis. Physical Stability and Viscosity Behavior of NISTmAb Reference Material in Various Excipient Conditions. XVII International Small Angle Scattering Conference - SAS2018 October 7-12, 2018, Traverse City, Michigan USA. <b>Poster</b></li>
        <li>A. McClusky, S. Parker, K.J. Edler. pylj: an open-source Python library for teaching the interaction between molecular simulation and scattering. XVII International Small Angle Scattering Conference - SAS2018 October 7-12, 2018, Traverse City, Michigan USA.  <b>IUCr Journals Prize for the Best Student Lecture, SAS2018</b></li>
        <li>A. McClusky, S. Parker, K. Edler. Using high-performance computing and molecular dynamics to rationalise micelle structure from small-angle scattering. XVII International Small Angle Scattering Conference - SAS2018 October 7-12, 2018, Traverse City, Michigan USA.</li>
        <li>S.J. Perkins, A.J. Osborne, R. Nan, A. Miller, J.S. Bhatt, J. Gor.  Different folded-back monomeric solution structures for complement factor H clarify distinct functions in the wilde-type TYR402 and disease-related HIS402 allotypes. XVII International Small Angle Scattering Conference - SAS2018 October 7-12, 2018, Traverse City, Michigan USA.</li>
		<li>G.K. Hui, J. Gor, J. Lu, P.D. Sun, S.J. Perkins. Atomistic X-ray solution scattering structures of human IgG1 and IgG4 antibody-receptro complexes. XVII International Small Angle Scattering Conference - SAS2018 October 7, 2018, Traverse City, Michigan USA.</li>
        <li>E. Brookes, M. Rocco, S. Krueger, J. Curtis. <a href="https://www.chess.cornell.edu/public/events/biosas-advanced-applications">Tutorials on US-SOMO and SASSIE web</a> presented at the BioSAS:Advanced Applications workshop at the XVII International Small Angle Scattering Conference - SAS2018 October 7-12, 2018, Traverse City, Michigan USA.</li>
        <li>Savelyev, A, Brookes, E. Plotting Advancements to the GenApp Framework. Gateways 2018 September 25-27, 2018, University of Texas, Austin, TX USA.</li>
        <li>N. Leung, E. Brookes, J. E. Curtis. JSSSE: An Extensible and Configurable Molecular Structure Editor for Scientific Gateways. Gateways 2018 September 25-27, 2018, University of Texas, Austin, TX USA. <b>Poster</b></li>
        <li>Y. Chen, A. Savelyev, E. Brookes, D. Fushman. ROTDIF-Web: A GenApp Generated Science Gateway for Comprehensive Analysis of Biomolecular NMR Relaxation Data.  Gateways 2018 September 25-27, 2018, University of Texas, Austin, TX USA. <b>Poster</b></li>
		<li>J. Wereszczynski - Exploring the Relationship Between the Structure, Function, and Dynamics of Biomolecular Complexes - Illinois Institute of Technology Department of Physics, September 2018</li>
        <li>A. McClusky, B. Morgan, S. Parker, K.J. Edler. Introducing programming to undergraduate chemists: and the tools we've developed to help them. PYCON UK, Cardiff, September 16, 2018.</li>
		<li>J S Bhatt. Molecular dynamics combined with SANS.  'Neutrons and Biology' School of the French Neutron Society, September 16-19, 2018, Carqueiranne, France</li>
		<li>Brookes, E.H. Definition File Generation of Science Gateways using GenApp, International Workshop on Science Gateways, IWSG18, 15 June 2018,  Edinburgh, UK.</li>
		<li>S.J.Perkins. Monte Carlo atomistic modelling of X-ray and neutron scattering data for human IgG1 and IgG4 antibodies reveals new insights on antibody solution structure and function. 3rd International Conference on Biosciences (ICB-2018) from 9-11th May, 2018, Science block, GC University Lahore Pakistan.</li>
		<li>S.J.Perkins. Monte Carlo atomistic modelling of X-ray and neutron scattering data for human IgG1 and IgG4 antibodies reveals new insights on antibody solution structure and function. National Institute for Biotechnology & Genetic Engineering, Faisalabad, Pakistan. May 10, 2018</li>
        <li>J. Wereszczynski - Exploring the Relationship Between the Structure, Function, and Dynamics of Biomolecular Complexes - Temple University Department of Physics, April 2018</li>
		<li>S.J.Perkins. The CCP-SAS project on atomistic modelling of SAS data. Neutrons and Global Challenges II: Health and Healthcare. Institute of Physics, Feb 9, 2018. London, UK.</li>
		<li>D.J. Scott. Neutron scattering with disordered proteins: soft matter techniques applied to hard biology. Neutrons and Global Challenges II: Health and Healthcare. Institute of Physics, Feb 9th 2018. London, UK.</li>

        <!-- 2017 -->
        <b>2017</b>
		<li>D.J. Scott. Intrinsic disorder in the KorB system. School of Vetinary and Medical Sciences, 15th Nov 2017. Nottingham, UK.</li>
		<li>K. Edler.  Polymer Effects on Formation of Polymer-Stabilized Lipid Nanodiscs. Nanoparticle Technologies for Membrane Protein Research, Jun 27, 2017. Leeds, UK.</li>
        <li>E. Brookes & A. Savelyev. GenApp Integrated with OpenStack Supports Elastic Computing on Jetstream. Practice & Experience in Advanced Research Computing Conference, Jul 9-13 2017. New Orleans, USA.</li>

        <!-- 2016 -->
        <b>2016</b>
        <li>A. Savelyev & E. Brookes. GenApp: Extensible Tool for Rapid Generation of Web and Native GUI Applications. Gateways 2016 Conference, Nov 3, 2016. San Diego, California.</li>
		<li>J. Chen. Towards Reliable Atomistic Simulation of Disordered Protein Ensembles. 252nd American Chemical Society Annual Meeting, Aug 21-25, 2016. Philadelphia, USA.</li>
        <li>K. Lee, J. Chen. Towards a Balanced Implicit Solvent Force Field for Intrinsically Disordered Proteins. 252nd American Chemical Society Annual Meeting, Aug 21-25, 2016. Philadelphia, USA.</li>
        <li>X. Liu, J. Chen. Coarse-grained model for multi-scale simulation enhanced sampling of intrinsically disordered protein conformations. 252nd American Chemical Society Annual Meeting,  Aug 21-25, 2016. Philadelphia, USA.</li>
        <li>E.Brookes, P. Vachette, J. Pérez, M. Rocco. Recent updates to the UltraScan SOlution MOdeler (US-SOMO) HPLC-SAXS data analysis module. ACA 2016, Jul 23, 2016. Denver, USA.</li>
        <li>S. Krueger, C. Castañeda, J. Curtis, D. Fushman. Linkage-Specific Conformational Ensembles of Polyubiquitin Chains Revealed by NMR, SANS and Ensemble Structure Modeling. ACA 2016, Jul 23, 2016. Denver, USA.</li>
        <li><a href="https://www.youtube.com/watch?v=g15QeR5-Xkk&index=40&list=PLYx7XA2nY5Gf37zYZMw6OqGFRPjB1jCy6">S.C. Howell, E. Brookes, J. Curtis. Using Python to Create an Integrated Modular Framework for Atomistic Modeling of Molecular Structures. Scientific Computing with Python. Jul 11-17, 2016. Austin, USA.</a> <i>(See 09:11-14:03 mins)</i></li>
        <li>J. Chen. Multi-scale modeling of IDP structure and interaction. 251st American Chemical Society National Meeting, Mar 13-17, 2016. San Diego, USA.</li>
        <li>S. Perkins. CCP-SAS ? a community consortium for the atomistic modelling of scattering data: example applications. International Conference on Molecular Recognition. Feb 1, 2016. Zaragoza, Spain.</li>

        <!-- 2015 -->
        <b>2015</b>
        <li>K. Edler. Polymer stabilized phospholipid nanodiscs. In Symposium ?Self-assembled Biofunctional Nanomaterials?, Pacifichem, Dec 15 ? 20, 2015, Honolulu, USA.</li>
		<li>K. W. Fung, R. Nan, M. Swann and S. J. Perkins. Dual polarization interferometry: characterisation of complement Factor H and C4b binding orientation to surfaces. Farfield and QCM-D User Meeting. Oct 26-28, 2015. Manchester University, UK.</li>
		<li>J. Curtis. Neutron Scattering and Simulation for Structural Biology and Biotechnology. Oct 21, 2015. Kansas State University, USA.</li>
        <li>J. Chen. Multi scale Simulations of IDP Structure and Interaction. 12th New England Structure Symposium,  Oct 10, 2015. Storrs, USA.</li>
        <li>E. Brookes, A. Kapoor, P. Patra, S. Marru, R.  Singh, M. Pierce. GSoC 2015 student contributions to GenApp and Airavata. 10th Gateway Computing Environments Workshop, Sep 30, 2015. Boulder, USA.</li>
		<li>J. Curtis. Software for Atomistic Modeling and Analysis of X-ray and Neutron Scattering Data. Sep 21, 2015. Max Planck Institute of Biophysics, Frankfurt am Main, Germany.</li>
        <li>W. Zhang, A. Heindel, D. Wright, J. Curtis, J. Chen. Advanced sampling and atomistic modeling for structural interpretation of small angle scattering. 16th International Conference on Small-Angle Scattering, Sep 13-18, 2015. Berlin, Germany.</li>
		<li><a href="/Resources/curtis_sas_2015_berlin_final.pdf">J. Curtis. CCP-SAS - A community consortium for the atomistic modelling of scattering data. 16th International Conference on Small-Angle Scattering, Sep 16, 2015. Berlin, Germany.</a></li>
		<li>S. Perkins. The asymmetric solution structures of native and patient monomeric human IgA1 reveal new insights on IgA nephropathy. 16th International Conference on Small-Angle Scattering, Sep 15, 2015. Berlin, Germany.</li>
		<li>J. Curtis & D. Wright. Live demonstration of SASSIE-Web. 16th International Conference on Small-Angle Scattering, Sep 14, 2015. Berlin, Germany.</li>
        <li>J. Chen. Towards Reliable Atomistic Simulations of Disordered Ensembles. CECAM Workshop on Intrinsically Disordered Proteins - Bringing together Physics, Computation and Biology, Aug 18-21, 2015. Zürich, Switzerland.</li>
		<li>H. Zhang, J. Curtis, E. Brookes. Massively Parallel Computation for Small Angle Scattering. 65th Annual Meeting of American Crystallography Association, Jul 28, 2015. Philadelphia, USA.</li>
		<li>S. Krueger. SANS Contrast Variation Experiments on Protein Complexes with Disordered Subunits. 65th Annual Meeting of American Crystallography Association, Jul 26, 2015. Philadelphia, USA.</li>
		<li>J. Curtis, S. Perkins, P. Butler, S. King, J. Chen, H. Zhang, D. Wright, E. Brookes. CCP-SAS: A Community Consortium for the Atomistic Modelling of Scattering Data. 65th Annual Meeting of American Crystallography Association, Jul 26, 2015. Philadelphia, USA.</li>
		<li>J. Curtis. SASSIE: A framework for multi-scale modeling of scattering data. EMBO Practical Course on Small angle neutron and X-ray scattering from proteins in solution, May 19, 2015. Grenoble, France.</li>
        <li><a href="https://www.youtube.com/watch?v=AEyWr7_8qBE">E. Brookes. Gateways Application Generation with GenApp. XSEDE Weekly Gateways Seminar, May 1, 2015.</a></li>
		<li>D. Wright, R. Nan, G.-K. Hui, J. Curtis, E. Brookes, S. Perkins. CCP-SAS - Novel approaches for the atomistic modelling of small angle scattering data in biology. 59th Biophysical Society Annual Meeting, Feb 7-11, 2015. Baltimore, USA.</li>
		<li>K. Lee, J. Chen. Refining Multi-scale Enhanced Sampling for Simulating Disordered Protein Conformations. 59th Biophysical Society Annual Meeting, Feb 7-11, 2015. Baltimore, USA.</li>
		<li>D. Wright, R. Nan, G.-K. Hui, J. Curtis, E. Brookes, S. Perkins. CCP-SAS - Novel approaches for the atomistic modelling of small angle scattering data in biology. CCP-Biosim Meeting. Jan 7-9, 2015. Leeds, UK.</li>

        <!-- 2014 -->
        <b>2014</b>
		<li>E. Brookes. An Open Extensible Application Generation Tool for Simple Rapid Deployment of Multi-Scale Scientific Codes. University of Texas at San Antonio, Computer Science Department talk. San Antonio, USA.</li> 
		<li>J. Chen. Orderly Chaos of Proteins. University of Kansas Medical Center, Biochemistry and Molecular Biology departmental seminar, Dec 19, 2014. Kansas City, USA.</li>
		<li>E. H. Brookes, N. Anjum, J. E. Curtis, S. Marru, R. Singh, M. Pierce. GenApp module execution and airavata integration. 9th Gateway Computing Environments Workshop, SC14, Nov 16-21, 2014. New Orleans, USA.
		<li>P. Butler. CCP-SAS ? a new Collaborative Computational Project for the atomistic or coarse grained modelling of SANS and SAXS data. Joint PSB/COLLEGE 8 Seminar, Oct 16, 2014. Grenoble, France.</li>
		<li>J. Chen. Multi-Scale Enhanced Sampling of Protein Structure and Interaction. Shanghai Jiao Tong University, Department of Physics seminar, Sept 22, 2014. Shanghai, China.</li>
		<li>E. Rodriguez. A revised mechanism for the activation of complement C3 to C3b and a molecular explanation of the difference between their C3S and C3F polymorphic forms. XXV International Complement Workshop, Rio de Janeiro, Sep 14-18, 2014.</li>
		<li>J. Chen. Multi-Scale Enhanced Sampling of Protein Structure and Interaction. Xiamen University, Department of Physics seminar, Sept 14, 2014. Xiamen, China.</li>
		<li>S. J. Perkins, G.-K. Hui, L. Rayner. Atomistic modelling of SANS and SAXS data in applications to diverse antibody classes important in biotechnology and disease. Neutron Characterisation in Fundamental and Applied Biotechnology (NCFAB). Sep 3-5, 2014. Abingdon, UK.</li>
		<li>J. Chen. Accelerate sampling in atomistic energy landscapes using coarse-grained models for implicit solvent optimization. 248th ACS National Meeting, Aug 10-14, 2014. San Francisco, USA.</li>
		<li>E. Brookes. An Open Extensible Multi-Target Application Generation Tool for Simple Rapid Deployment of Multi-Scale Scientific Codes. XSEDE 2014 Annual Conference on Extreme Science and Engineering Discovery Environment, Jul 13-18,. 2014. Atlanta, USA.</li>
		<li>S Perkins. Atomistic scattering modelling in CCP-SAS - its application to human IgG4. British Biophysical Society Biennial Meeting 2014, Jul 9-11, 2014. Warwick, UK.</li>
		<li>R. Nan, C. M. Furze, D. W. Wright, R. Wallis, S. J. Perkins. The solution structures of MASP (mannose-binding lectin-associated serine protease) and MBL (mannose-binding lectin) provides insight on the activation of the lectin pathway of complement. British Biophysical Society Biennial Meeting 2014, Jul 9-11, 2014. Warwick, UK.</li>
		<li>E. Brookes. Optimizing utilizable information from small angle solution scattering of biological macromolecules. British Biophysical Society Biennial Meeting 2014, Jul 9-11, 2014. Warwick, UK.</li>
		<li>E. Brookes, M. Rocco. UltraScan-SOMO (US-SOMO): An Integrated Hydrodynamic Calculation and Small Angle Scattering Data Analysis Software Suite. 3rd S4SAS Workshop, Jul 7-9, 2014. Didcot, UK.</li> 
		<li>J. Chen. Accelerate sampling in atomistic energy landscapes using topology-based coarse-grained models. Telluride Workshop on Advances in Enhanced Sampling Algorithms, Jun 30 ? Jul 4, 2014. Telluride, USA.</li>
		<li>S. Perkins. European Molecular Biophysics Facility Network Initiative.  5-6 June 2014. Institut Pasteur, Paris, France.</li>
		<li>H. Zhang. Simulation of Neutron Data of Intrinsically Disordered Proteins and Nucleic Acids: Part II. ACNS 2014, Jun 1-5, 2014. Knoxville, USA.</li>
		<li>J. E. Curtis. Introduction to Simulation Methods Applied to Modelling Neutron Scattering Structural Data. ACNS 2014, Jun 1-5, 2014. Knoxville, USA.</li>
		<li>J. Perez, M. Rocco, E. Brookes. Bio SAS Data Analysis within US-SOMO. ACA 2014, May 24-28, 2014. Albuquerque, USA.</li>
		<li>J. E. Curtis & H. Zhang. SASCALC -- A fast and accurate small-angle scattering calculator for atomistic ensembles. ACA 2014, May 24-28, 2014. Albuquerque, USA.</li>
		<li>P. Vachette, J. Perez, M. Rocco, E. Brookes. New developments in the UltraScan SOlution MOdeler (US-SOMO) HPLC-SAXS data analysis module. ACA 2014, May 24-28, 2014. Albuquerque, USA.</li>
		<li>E. Brookes. Progress in Parsimonious Spatial Modelling of Biological SAS Experimental Data. ACA 2014, May 24-28, 2014. Albuquerque, USA.</li>
		<li>P. Butler. CCP-SAS: A Collaborative Computational Project for Small Angle Scattering Modelling. SI2 PI workshop, Feb 24-25, 2014. Arlington VA.</li>
		<li>S. Perkins, D. Wright, J. Curtis. CCP-SAS ? a new community consortium for the atomistic modelling of SANS and SAXS data. Neutrons in Biology and Biotechnology, February 19-21, 2014. Grenoble, France.</li>
	</ul>
   </div>

</div>

<!-- THIS SHOULD NEVER BE TOUCHED -- at some point we should figure out how to add links in CSS 
     this can become an single line and the footer can be easily changed across the site (for 
	 example for adding a sponsor logo or text about sponsors/participants etc etc) -->        
<div id="footer">
	<div id="footerImages">
<!--
		<a id='CCPSAS simple logo' href="http://www.ccpsas.org"><img src="media/ccpsas_logo_simple_colour.png"></a>
-->
		<a id='EPSRC logo' href="http://www.epsrc.ac.uk/"><img src="media/epsrc_logo.png"></a>
		<img src="media/blank.png">
		<a id='CCPSAS simple logo' href="http://www.ccpsas.org"><img src="media/ccpsas_logo_detailed_colour.png"></a>
		<img src="media/blank.png">
		<a id='NSF logo' href="http://www.nsf.org"><img src="media/nsf_logo.png"></a>
	</div>
</div>

</body></html>
